descope/management/access_key.py

```diff
@@ -120,6 +123,7 @@ def update(
         self,
         id: str,
         name: str,
+        description: Optional[str] = "",
```

Comment:
In Python, the use of `Optional[str]` means that the variable can be of type `str` or `None`. When you assign a default value of "" (an empty string), you are indicating that the variable should always be a string, even if it’s an empty one. Therefore, in this context, it would be more appropriate to use `str` rather than `Optional[str]`.